scenarios/web/spring-boot/src/main/resources/templates/index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org" xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" layout:decorator="layout">
```

Comment:
Don't let the fact that this file still has a *container* `html` and `body` element throw you. This is a convention Thymeleaf does so that their views can be validated and interact with editors more gracefully. It doesn't affect runtime, this will work like a Razor view + Layout when rendered.